roles/internal/righttoknow/tasks/main.yml

```diff
@@ -107,28 +108,29 @@
   copy:
     content: "{{ ruby_version_staging }}"
     dest: /srv/www/staging/shared/rbenv-version
+  when: not (standalone_production | default(false))
```

Comment:
shall we clean up in a few weeks and make standalone the normal?

Reply:
Yep - that's the plan!